components/automate-ui/src/app/modules/infra-proxy/cookbook-details/cookbook-details.component.ts

```diff
@@ -138,6 +139,15 @@ export class CookbookDetailsComponent implements OnInit, OnDestroy {
         this.menuList = CollapsibleListMapper.transform(cookbookDetailsState, this.listItem);
         if (this.menuList.length) {
           // check submenu exists and show default item details
+          for (const menu of this.menuList) {
+            if ( menu.subMenu.length ) {
+              this.defaultContent = menu.subMenu[0];
+              this.getContent(this.defaultContent);
+              this.menuIndex = this.menuList.indexOf(menu);
+              break;
+            }
+          }
+
```

Comment:
Suggest chaining and arrow functions when possible in keeping with current Chef convention
```
const submenus = this.menuList.filter(menu => menu.subMenu.length);

submenus.forEach(menu => {
    this.defaultContent = menu.subMenu[0];
    this.getContent(this.defaultContent);
    this.menuIndex = this.menuList.indexOf(menu);
});
```

Reply:
Thank you for review @SEAjamieD 
Actually, your suggested code of block targeting the last menu, and our requirement is to target the first menu which has contained some submenu.
after we got the first menu (which has some submenu) I am using `break` to stop executing the remaining menus.

or we can use
```
const menus = this.menuList.filter(menu => menu.subMenu.length);
if ( menus.length ) {
   this.defaultContent = menus[0].subMenu[0];
   this.getContent(this.defaultContent);
   this.menuIndex = this.menuList.indexOf(menus[0]);
}
```

![Chef Automate (13)](https://user-images.githubusercontent.com/12297653/113982665-a8bd5a80-9866-11eb-9655-7b70d540cc2c.png)

Reply:
Good follow through - thanks @vinay033.  My suggestion would be to adjust in the most appropriate way that allows us to still follow prior conventions, so if you can do this with arrow functions and chains, that would be ideal.

Reply:
I have updated the PR, please check it once.